charts/openproject/templates/_helpers.tpl

```diff
@@ -37,11 +37,46 @@ securityContext:
 {{- end }}
 
 {{- define "openproject.useTmpVolumes" -}}
-{{- if (default .Values.useTmpVolumes (not .Values.develop)) -}}
+{{- $useTmpVolumes := .Values.openproject.useTmpVolumes | toString | lower }}
```

Comment:
Why do we need to stringify this and wouldn't this mean we need to do this for all kinds of other values too, like `persistence.enabled`, `s3.enabled` etc..